src/app/metrics/site-metrics/site-metrics.component.html

```diff
@@ -0,0 +1,63 @@
+<div class="container-fluid">
+  <h1>Site Metrics</h1>
+  <hr />
+  <app-metrics-filter></app-metrics-filter>
+  <div>
+    <!-- Loading element when API ready -->
+    <div>
+      <h2>Data for [Park name]: [Facility Name]</h2>
+      <div class="container-fluid p-0">
+        <div class="row d-flex justify-content-center">
+          <div
+            class="col-12 col-md-4 pl-sm-2 row flex-column justify-content-between col-height"
+          >
+            <app-metric-card
+              class="metric-card"
+              [title]="'Total Number of Passes'"
+            >
+              {{ passTotal }}
```

Comment:
`<app-basic-metric>` was created to display/animated the numerical metric values, we should be passing `passTotal` into this component within each `<app-metric-card>`

Reply:
Properly changed things to wrap `<app-basic-metric>` with `<app-metric-cards>`